modules/bookAdditon.js

```diff
@@ -0,0 +1,89 @@
+class SavedObjects {
+  constructor() {
+    this.objects = [];
+    this.objectsContainer = document.getElementById('objectsContainer');
+    this.submitButton = document.getElementById('submitBtn');
+    this.titleInput = document.getElementById('title');
+    this.descriptionInput = document.getElementById('description');
+    this.successMessage = document.getElementById('successMessage');
+
+    this.submitButton.addEventListener('click', this.handleSubmit.bind(this));
+  }
+
+  addObject(object) {
+    this.objects.push(object);
+    localStorage.setItem('savedObjects', JSON.stringify(this.objects));
+  }
+
+  retrieveObjects() {
+    const savedData = localStorage.getItem('savedObjects');
+    if (savedData) {
+      this.objects = JSON.parse(savedData);
+    }
+  }
+
+  displayObjects() {
+    this.objectsContainer.innerHTML = '';
+
+    this.objects.forEach((object, index) => {
+      const objectElement = document.createElement('div');
+
+      const titleElement = document.createElement('span');
+      titleElement.textContent = `${object.title}...by `;
+
+      const descriptionElement = document.createElement('span');
+      descriptionElement.textContent = `...${object.description}`;
+
+      const removeButton = document.createElement('input');
+      removeButton.type = 'button';
+      removeButton.value = 'Remove';
+      removeButton.addEventListener('click', () => {
+        this.removeObject(index);
+        this.displayObjects();
+      });
+
+      objectElement.appendChild(titleElement);
+      objectElement.appendChild(descriptionElement);
+      objectElement.appendChild(removeButton);
+
+      this.objectsContainer.appendChild(objectElement);
+    });
+  }
+
+  removeObject(index) {
+    this.objects.splice(index, 1);
+    localStorage.setItem('savedObjects', JSON.stringify(this.objects));
+  }
```

Comment:
- [ ] Please ensure that you use arrow function syntax as opposed to regular function as part of the project requirement. 

A possible solution would be: :arrow_down: 

` removeObject = (index) => {`
    `this.objects.splice(index, 1);`
   ` localStorage.setItem('savedObjects', JSON.stringify(this.objects));`
`  }`


- [ ] Please ensure that you replace most of your regular functions with Es6 arrow syntax in all your js files as this is some of the essence of this particular milestone. :slightly_smiling_face: